### DISCUSSION TITLE
In which mode the Disch pin is?

### DISCUSSION THREAD

**Discussion by otsatukka** [oxplot/fpx#26] [Q&A]
Can the disch pin be connected directly to the GND?  In the stusb4500 datasheet the min connection shows a series connection to the vbus line after the transistor. In which mode the disch pin is? From datasheet :

> _When used as input, the discharge is internal and a serial resistor must connected to the pin to limit the discharge
> current through the pin. Maximum discharge current is 500 mA.
> The pin can be also used as an open drain output to control an external VBUS discharge path when higher
> discharge current is required by the application, for instance._

Is it output or input?

I'm trying to take the stusb4500 also in use and would value input for this question.

Thanking in advance
Mikko

**Comment by oxplot** [comment]
The discharge pin is "used" to sink the charge on the VBUS line into the chip via the resistor so its max current is limited (max current is the max voltage (ie 20v) divided by the value of the resistor).

Note that the discharge pin has a single behavior — in other words, there isn't a mode you can configure on the chip. It's how you "use" it that makes it act like an input/output. The discharge pin is an active low pin. When disabled by the chip, it's floating (ie very high resistance). When enabled, it's pulled to ground. You can use this behavior to either directly discharge the VBUS line into the chip, OR have the discharge pin pull the gate of a P-Channel MOSFET to ground in order to turn on the MOSFET and discharge a higher current to ground.

**Comment by oxplot** [comment]
Yes, the discharge pin can be connected to ground or simply left floating.

**Comment by otsatukka** [comment]
Thank you for your rapid answer! 

Does connecting it to ground or leaving it floating then mean that this discharge path is not used at all and the VBUS_VS_DISCH is the only discharge path? Maybe I still missed something...

Thanks anyway I able to continue with this
